pages/index.tsx

```diff
@@ -11,7 +11,7 @@ import Section from '../components/Section'
 import TextBlock from '../components/TextBlock'
 import Tuber from '../components/Tuber'
```

Comment:
Rule, Section, TextBlock, and Tuber should be nested in Main. Otherwise, perfect.

Reply:
Whoops. I appear to be logged in as Jake. But this is Chas.